libs/ui-lib/lib/common/types/cpuArchitecture.ts

```diff
@@ -77,17 +77,11 @@ export const getSupportedCpuArchitectures = (
 ): SupportedCpuArchitecture[] => {
   const newSupportedCpuArchs: SupportedCpuArchitecture[] = [];
   //Power/Z clusters can be only homogeneous clusters
```

Comment:
I think the comment needs to be updated, now only Power clusters would be homogeneous.